### PR TITLE
AS-917: Switch ToS endpoints to use Sam [DNM until ToS is enabled]

### PR DESCRIPTION
**Pull request by MatthewBemis**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/endpoints.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/endpoints.cljs:584]: can `config/tos-application-name` and `config/tos-version` now be deleted?

File: src/cljs/main/broadfcui/auth.cljs
Comment (davidangb) [inline src/cljs/main/broadfcui/auth.cljs]: does this do anything on the back end? What does sending an intentionally incorrect tos url to Sam actually do?

**Review by davidangb — COMMENTED** [review]
two questions inline, looks good otherwise. I did not test!

**Review by davidangb — COMMENTED** [review]
two questions inline, looks good otherwise. I did not test!

File: src/cljs/main/broadfcui/auth.cljs
Comment (MatthewBemis) [reply src/cljs/main/broadfcui/auth.cljs]: That's a good point, this should be a no-op

**Comment by MatthewBemis** [comment]
@davidangb this should be all set for re-review now

**Review by davidangb — APPROVED** [review]
lgtm! As you noted, can't merge this until the ToS is enabled.

Sometime after this merges, we'll need a followup to remove the ToS keys from firecloud-develop: https://github.com/broadinstitute/firecloud-develop/blob/a88217b2c921d7405c374f8139dc3feaf33a9772/base-configs/firecloud-ui/config.json.ctmpl#L33-L34

**Review by sagehen03 — APPROVED** [review]

**Pull request closed without merging**
